api/kyverno/v1/common_types.go

```diff
@@ -307,7 +308,23 @@ type Validation struct {
 
 	// Deny defines conditions used to pass or fail a validation rule.
 	// +optional
-	Deny *Deny `json:"deny,omitempty" yaml:"deny,omitempty"`
+	Deny        *Deny        `json:"deny,omitempty" yaml:"deny,omitempty"`
+	PodSecurity *PodSecurity `json:"podSecurity,omitempty" yaml:"podSecurity,omitempty"`
```

Comment:
If a field is optional, you need to add the tag `// +optional`. Otherwise the policy will be blocked if the field is not present.